app/controllers/questionnaire_controller.rb

```diff
@@ -207,7 +207,7 @@ def store_response_cookie
   end
 
   def set_is_mentor
-    @use_mentor_layout = @response.protocol_subscription.person.mentor?
+    @use_mentor_layout = @response.protocol_subscription.person.mentor? || @response.protocol_subscription.person.solo?
```

Comment:
misschien even voor de duidelijkheid deze functie ook `set_layout` noemen.

Reply:
Hieruit blijkt btw ook weer dat er iets nog niet helemaal lekker loopt met die layout. Ik denk eigenlijk dat we dit stuk ook moeten fixen op het moment dat we de applicatie helemaal los gaan trekken (de seeds eruit e.d.)

Reply:
Ja dit zou je ook gewoon uit de vragenlijstdefinitie kunnen lezen. Het enige waar je dan nog mee zit is de people_controller (dwz de /person/edit pagina), want die hangt niet altijd aan een vragenlijst vast dus het is moeilijk te zeggen of iemand een student of mentor is.

Fixed.